template/src/index.js

```diff
@@ -1,4 +1,5 @@
 import './main.css'
+import './Stylesheets.elm'
```

Comment:
Ok, sounds reasonable. Should I also delete the file `Stylesheets.elm` and instead provide an example on how to create this file in the user guide?